pvlib/pvsystem.py

```diff
@@ -534,11 +534,11 @@ def retrieve_sam(name=None, samfile=None):
         name = name.lower()
 
         if name == 'cecmod':
-            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-cec-modules-2014-1-14.csv'
+            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-cec-modules-2015-6-30.csv'
         elif name == 'sandiamod':
-            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-sandia-modules-2014-1-14.csv'
-        elif name == 'sandiainverter':
-            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-sandia-inverters-2014-1-14.csv'
+            url = 'https://sam.nrel.gov/sites/sam.nrel.gov/files/sam-library-sandia-modules-2015-6-30.csv'
+        elif name == 'sandiainverter': # Still only one current inverter set, so left it as sandia, to avoid breaking old code
```

Comment:
What if we do something like ``elif 'inverter' in name`` or ``elif name in ['sandiainverter', 'cecinverter']`` to cover both the old code and the expectations of new users?